cibuildwheel/resources/pinned_docker_images.cfg

```diff
@@ -19,4 +19,3 @@ manylinux2014 = quay.io/pypa/manylinux2014_ppc64le:2020-05-15-5acda28
 
 [s390x]
 manylinux2014 = quay.io/pypa/manylinux2014_s390x:2020-05-15-5acda28
-
```

Comment:
This is generated. Then we need to fix the script.

Reply:
Agreed. This file is generated in `bin/update_dependencies.py`. I don't see an option to disable this last newline in `configparser`. Perhaps this hook could be removed also?